test/1_stdlib/PrintForDebugger.swift

```diff
@@ -0,0 +1,51 @@
+// RUN: %target-run-simple-swift | FileCheck %s
+// REQUIRES: executable_test
+
+struct StructWithMembers {
+  var a = 1
+  var b = "Hello World"
+}
+
+print(_PrintForDebugger.printForDebugger(value: StructWithMembers()))
+// CHECK: StructWithMembers
```

Comment:
The advantage is better debuggability, better crash reporting, and less fuzzy matching.